pixart.js

```diff
@@ -1 +1,13 @@
 
+$(document).on("ready", function() {
+    console.log('Ready');
+
+    $('button').on("click", userInput);
+
+ function userInput(event) {
+   event.preventDefault();
+   var setColor = $('#color-field').val();
+   $('.brush').css("background-color", setColor);
+ }
+
+ });
```

Comment:
🔨 Nailed it. The only minor recommendation I have is to name userInput something more indicative of what the function does like changeColorOnClick or something. Great job Alex!